src/test/cypress/support/requests/CourseManagementRequests.ts

```diff
@@ -630,3 +633,12 @@ export enum CypressExerciseType {
     TEXT,
     QUIZ,
 }
+
+export function parseCourseAfterMultiPart(response: Cypress.Response<Course>): Course {
+    // Cypress currently has some issues with our multipart request, parsing this not as an object but as an ArrayBuffer
+    // Once this is fixed (and hence the expect statements below fail), we can remove the additional parsing
+    expect(response.body).not.to.be.an('object');
+    expect(response.body).to.be.an('ArrayBuffer');
```

Comment:
Isn't it automatically not an 'object' when it's an ArrayBuffer?

Reply:
Yes, it's meant as help for the person looking at that section once that fails due to a Cypress fix. Both statements would be removed then